components/progress-rope/src/ProgressRopeGroup.js

```diff
@@ -0,0 +1,79 @@
+/** @jsx jsx */
+
+import { Children, cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import { jsx, useBrand } from '@westpac/core';
+import { useProgressRopeContext } from './ProgressRope';
+
+export const ProgressRopeGroup = ({ index, label, children, ...props }) => {
+	const { openGroup, ropeGraph, handleClick } = useProgressRopeContext();
```

Comment:
We may not need context here?

Reply:
Why not? I'm using those values calculated in the parent ProgressRope component

Reply:
Can't just use props? Too many?

Reply:
Im a bit confused here. I would have to clone the child component to pass props to it? I thought we were moving away from that pattern. 
```
<ProgressRope current={indexOfStep}>
   <ProgressRopeGroup>
      <ProgressRopeItem>Step</ProgressRopeItem>
   </ProgressRopeGroup>
</ProgressRope>
```
This is how I've structured the usage of the component. The values I'm grabbing via context are calculated in the ProgressRope component based on the index of the step that is passed in via props.

Reply:
I'll have a proper look tomorrow. No blockers from me. Happy to merge for now